File: app/src/main/java/com/ufpb/getha/ui/gallery/CalculadoraViewModel.java
package com.ufpb.getha.ui.gallery;

import androidx.lifecycle.LiveData;
import androidx.lifecycle.MutableLiveData;
import androidx.lifecycle.ViewModel;

public class CalculadoraViewModel extends ViewModel {

    private final MutableLiveData<String> mText;

    public CalculadoraViewModel() {
        mText = new MutableLiveData<>();
        mText.setValue("This is gallery fragment");
    }

    public LiveData<String> getText() {
        return mText;
    }
}

File: app/src/main/java/com/ufpb/getha/ui/gallery/CalculadoraViewModel.kt
package com.ufpb.getha.ui.gallery

import androidx.lifecycle.LiveData
import androidx.lifecycle.MutableLiveData
import androidx.lifecycle.ViewModel

class CalculadoraViewModel : ViewModel() {
    private val mText: MutableLiveData<String>

    init {
        mText = MutableLiveData()
        mText.value = "TODO"
    }

    val text: LiveData<String>
        get() = mText
}